Makefile

```diff
@@ -46,9 +47,24 @@ test: $(CONFIG)
 fast_test:
 	bundle exec rspec --exclude-pattern "**/features/accessibility/*_spec.rb"
 
+tmp/localhost.key tmp/localhost.crt:
+	mkdir -p tmp
+	openssl req \
+		-newkey rsa:2048 \
+		-x509 \
+		-sha256 \
+		-nodes \
+		-days 365 \
+		-subj "/C=US/ST=District of Columbia/L=Washington/O=GSA/OU=Login.gov/CN=localhost:3000"  \
```

Comment:
should we dynamically inject the port into this? maybe into the filename too so we can re-create if needed?

Reply:
> should we dynamically inject the port into this? maybe into the filename too so we can re-create if needed?

As far as I've been able to tell, the "Common Name" (CN) doesn't really seem to make much difference. Or, at least, I've been able to bind this in a few different formats with the hard-coded `localhost` value, so... maybe it's fine as static?

Reply:
Oh, or do you mean by the `PORT` variable we have at the top of the file? I suppose that can make sense, yeah.

Reply:
Using variables in ac8a66a.

Reply:
yes yes $HOST and $PORT. thank you!